Update FAQ.md (#10366)
iptv-org/iptv: FAQ.md

Based on [#1797](https://github.com/iptv-org/iptv/issues/1797), [#2438](https://github.com/iptv-org/iptv/issues/2438), [#4537](https://github.com/iptv-org/iptv/discussions/4537) and [#7043](https://github.com/iptv-org/iptv/discussions/7043).

### Is it possible to view channels directly from the main website?

> No. In own website you will only see an extensive list of channels and their respective information, like a global database. However, you will be able to search and review the channels you want, and if your favorite channel is legally available as an indexed stream in the IPTV playlist.

Reviewer: ```suggestion
No. In our website, you will only see an extensive list of channels and their respective information, like a global database. In this case, you will only be able to search and review the channels you want. If your favorite channel is legally available as an indexed stream in the IPTV playlist, there's a "Streams" option that will be available on the right side of the channel, so that you can have the link to watch it. Else, you can make an issue to ask for it. Please respect the [contribution guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md) before doing so;
```

Author: Thank you for improving the explanation.

### Is it possible to add streams from Youtube, Dailymotion or Twitch?
Yes. Due to technical limitations, it is necessary to add a [Streamlink](https://streamlink.github.io/) server link to view the content. This does not work with all streams, so it will give an "error" status if this stream is not freely accessible.

> The scheme for adding a YouTube stream is like this:

Reviewer: ```suggestion
As an example, here's the scheme for adding a YouTube stream:
```
